data/sql/convert_libpostal_yamls_to_sql_stdin.py

```diff
@@ -18,3 +19,18 @@
             line = '\t'.join([country_code, admin_level, mapped_type])
 
             print(line)
+
+print("")
```

Comment:
I've a little trouble to remember this script😅 one runs this script manually with the yaml files of libpostal and copies the output to the `admin_level_type_mapping.sql`, right?

Reply:
Yes, that's correct.